firefox-ios/Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetViewModel.swift

```diff
@@ -224,8 +224,7 @@ class PhotonActionSheetViewModel: FeatureFlaggable {
     private func getSmallSizeMargins(view: UIView, presentedOn viewController: UIViewController) -> UIEdgeInsets {
         // Align menu icons with popover icons
         let extraLandscapeSpacing: CGFloat = UIWindow.isLandscape ? 10 : 0
-        let isToolbarRefactorEnabled = featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly)
-        let statusIconSize = isToolbarRefactorEnabled ? 0 : PhotonActionSheetView.UX.StatusIconSize.width
+        let statusIconSize = 0.0
         let halfFrameWidth = view.frame.size.width / 2
         let rightInset = halfFrameWidth - PhotonActionSheet.UX.spacing - statusIconSize / 2 + extraLandscapeSpacing
```

Comment:
If its 0 let's remove it
```suggestion
        let halfFrameWidth = view.frame.size.width / 2
        let rightInset = halfFrameWidth - PhotonActionSheet.UX.spacing / 2 + extraLandscapeSpacing
```